Tools/YamlCreate.ps1

```diff
@@ -904,7 +904,7 @@ Function Submit-Manifest {
         git checkout -b "$PackageIdentifier-$PackageVersion" FETCH_HEAD
 
         git add -A
-        git commit -m "$CommitType : $PackageIdentifier version $PackageVersion"
+        git commit -m "$CommitType: `$PackageIdentifier` ver. $PackageVersion"
```

Comment:
I would suggest we stay with `version` to match wingetcreate. Commit messages usually are a matter of personal preference anyways, so long as they are understandable and accurate. For most of my commits I like to use `v1.2.3.0`, but there really isn't a difference if it were spelled out
```suggestion
        git commit -m "$CommitType: `$PackageIdentifier` version $PackageVersion"
```